src/cli/cli.cpp

```diff
@@ -3263,6 +3263,27 @@ otError Interpreter::ProcessPing(uint8_t aArgsLength, char *aArgs[])
 
     memset(&config, 0, sizeof(config));
 
+    if (aArgsLength >= 2)
+    {
+        if (!strcmp(aArgs[0], "-I"))
+        {
+            SuccessOrExit(error = ParseAsIp6Address(aArgs[1], config.mSource));
+            bool                  valid        = false;
```

Comment:
Should put all variables at the beginning of the scope.

See: https://github.com/openthread/openthread/blob/main/STYLE_GUIDE.md#white-space